### PR TITLE
History button

### PR DESCRIPTION
**Pull request by SergeyTverye**
Based on Alexander's remarks:

- UI moved to components
- JSX replaced with JS so that nobody worries about it
- OperationsList has been changed to tabular appearance 
- Made exports to the standard view

From the prev request:

- Moved UI pieces from the Operations component into separate small components to make the code more readable and less monolithic 

OperationsList can be easily moved to a separate button at any time, I suggest to do it after merge with navbar.

### CODE REVIEW

File: src/components/Items/OperationsList.js [hunk 1]
```diff
@@ -1,57 +1,63 @@
...
+import Table from "@mui/material/Table";
+import TableBody from "@mui/material/TableBody";
+import TableCell from "@mui/material/TableCell";
+import TableContainer from "@mui/material/TableContainer";
+import TableHead from "@mui/material/TableHead";
+import TableRow from "@mui/material/TableRow";
 import Typography from "@mui/material/Typography";
-import { light } from "@mui/material/styles/createPalette";
 
 function OperationsList({ operations }) {
```
Comment (SokolovAl) [inline src/components/Items/OperationsList.js]: Используй запись типа:
export const YourFunction = () => {}

File: src/routes/Operations.js [hunk 2]
```diff
@@ -1,19 +1,19 @@
...
-import Button from "@mui/material/Button";
-import Select from "@mui/material/Select";
-import MenuItem from "@mui/material/MenuItem";
-import Typography from "@mui/material/Typography";
 import {useActives} from "../hooks/useActives";
 import {useOperations} from "../hooks/useOperations";
 import {useUserPreference} from "../hooks/useUserPreference";
 import OperationsList from "../components/Items/OperationsList";
 import Stack from "@mui/material/Stack";
+//UI components
```
Comment (SokolovAl) [inline src/routes/Operations.js]: Можно убрать комментарий

**Review by SokolovAl — APPROVED** [review]
Только небольшие замечания по поводу стиля

File: src/components/UI/ToggleButtons.js
Comment (BorisZelikman) [inline src/components/UI/ToggleButtons.js:1]: History не нужна

**Review by BorisZelikman — APPROVED** [review]

**Pull request merged**